zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -955,37 +952,46 @@ public void emptyTrash(ServiceContext context,
   }
 
   public void spell(String noteId,
-                    Message message,
+                    String paragraphId,
+                    String title,
+                    String text,
+                    Object results,
+                    String errorMessage,
+                    String dateStarted,
+                    String dateFinished,
+                    String status,
```

Comment:
should this be typed Job.Status?

Reply:
Object type or a variable name?

Reply:
I think it should be Object, in case if we can't parse status, it may be set with a default value or  be gotten from another source

Reply:
if you want I could change the type of status.